reactive/flannel.py

```diff
@@ -245,6 +248,13 @@ def cleanup_deployment():
 
 def get_flannel_subnet():
     ''' Returns the flannel subnet reserved for this unit '''
-    with open('/run/flannel/subnet.env') as f:
-        raw_data = dict(line.strip().split('=') for line in f)
-    return raw_data['FLANNEL_SUBNET']
+    try:
+        with open('/run/flannel/subnet.env') as f:
+            raw_data = dict(line.strip().split('=') for line in f)
+        return raw_data['FLANNEL_SUBNET']
+    except FileNotFoundError as e:
+        raise FlannelSubnetNotFound() from e
```

Comment:
pardon my ignorance, by why catch a FileNotFound and then raise a pass wrapped by a function?